bin/run-ansible.sh

```diff
@@ -328,6 +334,11 @@ echo " |   Installing needed packages  |"
 echo " ---------------------------------"
 echo ""
 
+if [[ "${test_machine}" = "true" ]]; then
+    echo $test_password | sudo -S echo "Running on testing machine. Retrieving passwords from ENV..."
```

Comment:
Where does the variable `test_password` get set? Should we add an example or instructions or a script that calls run-ansible.sh after setting this variable?